packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -297,22 +294,18 @@ class Dart2WasmTarget extends Dart2WebTarget {
       final File optimizedOutput = environment.buildDir.childFile('main.dart.wasm');
       final List<String> optimizeArgs = <String>[
         wasmOptBinary,
-        '-all',
+        '--all-features',
```

Comment:
being explicit with all flags so they are easier to parse in source, debug logs, etc